ares/n64/cpu/cpu.hpp

```diff
@@ -277,17 +279,24 @@ struct CPU : Thread {
   auto userSegment64(u64 vaddr) const -> Context::Segment;
 
   auto segment(u64 vaddr) -> Context::Segment;
+  template<u32 Dir>
   auto devirtualize(u64 vaddr, bool raiseExceptions = true) -> PhysAccess;
   alwaysinline auto devirtualizeFast(u64 vaddr) -> u64;
   auto devirtualizeDebug(u64 vaddr) -> u64;
 
-  auto fetch(u64 vaddr) -> maybe<u32>;
+  auto fetch(PhysAccess access) -> maybe<u32>;
   template<u32 Size> auto busWrite(u32 address, u64 data) -> void;
   template<u32 Size> auto busRead(u32 address) -> u64;
   template<u32 Size> auto busWriteBurst(u32 address, u32 *data) -> void;
   template<u32 Size> auto busReadBurst(u32 address, u32 *data) -> void;
-  template<u32 Size> auto read(u64 vaddr) -> maybe<u64>;
-  template<u32 Size> auto write(u64 vaddr, u64 data, bool alignedError=true) -> bool;
+  template<u32 Size> auto read(PhysAccess access) -> maybe<u64>;
+  template<u32 Size> auto write(PhysAccess access, u64 data, bool alignedError=true) -> bool;
```

Comment:
Nit: spaces around '=' here and a few lines below

Reply:
ok